server/util/live-loan/live-loan-draw.js

```diff
@@ -1,32 +1,45 @@
+const { mdiMapMarker } = require('@mdi/js');
 const path = require('path');
-const { createCanvas, registerFont, loadImage } = require('canvas');
+const {
+	createCanvas,
+	CanvasRenderingContext2D,
+	registerFont,
+	loadImage,
+} = require('canvas');
 const deePool = require('deepool');
-const { ellipsisLine, wrapText, roundRect } = require('./canvas-utils');
+const numeral = require('numeral');
+const { polyfillPath2D } = require('path2d-polyfill');
+const {
+	ellipsisLine,
+	drawPill,
+	wrapText,
+	roundRect,
+} = require('./canvas-utils');
+const getLoanCallouts = require('../../../src/util/loanCallouts');
 const getLoanUse = require('../../../src/util/loanUse');
 const tracer = require('../ddTrace');
 
+// Polyfill Path2D for material design icon support
+global.CanvasRenderingContext2D = CanvasRenderingContext2D;
+polyfillPath2D(global);
+
+// Legacy styling constants
 const resizeFactor = 3;
-const borderThickness = 2 * resizeFactor;
 const cardWidth = 300 * resizeFactor;
 const cardHeight = 525 * resizeFactor;
-const bodyWidth = cardWidth * 0.85;
-const borrowerImgAspectRatio = 0.75;
-const kivaColors = {
-	action: '#2B7C5F',
-	brand: '#2AA967',
-	textLight: '#999999',
-	strokeGrey: '#d8d8d8',
-	textDark: '#484848',
-	white: '#ffffff'
-};
+
+// Kiva Classic styling constants
+const classicResizeFactor = 3;
+const classicCardWidth = 440 * classicResizeFactor;
+const classicCardHeight = 510 * classicResizeFactor;
 
 function fontFile(name) {
 	return path.join(__dirname, './fonts', name);
 }
 
 tracer.trace('registerFonts', () => {
 	/* eslint-disable max-len */
-	// registerFont(fontFile('PostGrotesk-Light.ttf'), { family: 'Kiva Post Grot', weight: '300' });
+	registerFont(fontFile('PostGrotesk-Light.ttf'), { family: 'Kiva Post Grot', weight: '300' });
```

Comment:
This font weight isn't actually being used, but when this line is commented out none of the other font weights load at all. It's also only this line that has that effect for me; no other lines being uncommented or commented made any difference to the loading behavior. It appears to be due to some quirk of the underlying OS that the server is running on, as this issue is happening on my local MacOS but is not happening on our Linux servers.
It doesn't hurt anything to leave this uncommented since all the font files are already on the server, so this doesn't use any additional resources.

Reply:
When I worked with canvas and custom fonts, it took quite a few iterations to get the fonts loaded async before the rendering started, so not surprising to see the differences. Good to hear it's not an issue on the servers.